src/Microsoft.AspNet.Mvc.Core/FileResult.cs

```diff
@@ -88,7 +88,7 @@ public override Task ExecuteResultAsync(ActionContext context)
                 context.HttpContext.Response.Headers[HeaderNames.ContentDisposition] = contentDisposition.ToString();
             }
             
-            logger.FileResultExecuting(FileDownloadName);
+            logger?.FileResultExecuting(FileDownloadName);
```

Comment:
If `CreateLogger()` can return `null`, many classes in the MVC repo will fall over. So I hope this isn't for a real scenario.